Topics/math-fundamentals.md

```diff
@@ -0,0 +1,172 @@
+# Math Fundamentals
+
+# What you will learn 
+In this guide, you'll learn what is the fundamental mathematical knowledge you need to have in order
+to build insightful Data Science projects and build machine learning systems upon them.
+
+You'll also learn that not so much is needed to start to apply yourself to real-world data problems!
+
+## Prerequisites
+The previous Paradiso guides.
+
+## Time to complete
+10 minutes.
+
+# Index
+- [Motivation](#Motivation)
+- [Courses](#Courses)
+  - [Calculus](#Calculus)
+  - [Linear Algebra and Matrix Algebra](#Linear-Algebra)
+  - [Optimization](#Optimization)
+- [Ask Questions](#Ask-Questions)
+- [Conclusions](#Conclusions)
+
+Let's dive right in!
+ 
+### Motivation
+
+Before anything else, watch [this](https://www.youtube.com/watch?v=VIbjHIGMjQM
+) and [this](https://www.youtube.com/watch?v=8CX-Q0gtSp8) awesome Ted Talks :-)
+
+**The entire field of Data Science is based on mathematics and statistics.**
+
+This phrase shouldn't frighten you, though, because as you'll find out, you probably have much of the knowledge you need to start experimenting!
+
+**In fact:** 
+
+If you've done a second-degree school (you know matrices and derivatives, for example), you probably just need to refresh yourself on the things you already know and go deeper into some things you may not have seen at school. 
+
+In general, if you have a scientific degree (engineering, computer science, physics, etc.), it will probably be just a review, as it will be the Calculus, Linear Algebra, and Matrix Algebra courses are suggested within this guide.
```

Comment:
> ....if you have done a science degree...

not sure if you can call a degree scientific - it could mean something else

Reply:
> it will probably be just a review, as it will be the Calculus, Linear Algebra, and Matrix Algebra courses are suggested within this guide.

minor: I suggest to rewrite this sentence a bit, it's not clear - although the problem

Reply:
fixed: scientific oriented 
fixed: re-written